includes/api/class-wc-admin-rest-onboarding-profile-controller.php

```diff
@@ -274,6 +274,21 @@ public static function get_profile_properties() {
 					'brick-mortar-other',
 				),
 			),
+			'revenue'      => array(
+				'type'              => 'string',
+				'description'       => __( 'Current annual revenue of the store.', 'woocommerce-admin' ),
+				'context'           => array( 'view' ),
+				'readonly'          => true,
+				'validate_callback' => 'rest_validate_request_arg',
+				'enum'              => array(
```

Comment:
One small issue here, but we might need to tackle in a follow-up- this actual value may vary greatly from currency to currency and in many currencies might be a very small amount.

I wonder if there are any packages that can convert to rough approximations/ranges based on currency.

Reply:
Really good point.

I've logged a follow-up issue so we can think about this a bit.

https://github.com/woocommerce/woocommerce-admin/issues/2768